src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -266,7 +266,8 @@ public void afterNavigateTo(String url, WebDriver driver) {
       User user = method.getAnnotation(Execute.class).asUser();
       if (user == User.ANONYMOUS) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS00108)